Sources/OutputTarget.swift

```diff
@@ -32,7 +32,7 @@ import Darwin.C
 
 private func getEnvValue(key: String) -> String? {
     let value = getenv(key)
-    return value != nil ? String.fromCString(value) : nil
+    return value != nil ? String(cString: value!) : nil
```

Comment:
Exactly. It will appreciated if you could make this change. The `guard let` way is preferred for a better readability and safety. Thank you!